.github/workflows/run_integration_tests.yml

```diff
@@ -9,6 +9,7 @@ on:
       solutions:
         type: choice
         description: Which test solutions (IntegrationTests, UnboundedIntegrationTests, or both) to run
+        required: false
```

Comment:
I'm curious why you added the `required: false` here.  The solutions var needs to be set one way or another (either to 'ALL' to run everything or to a specific sublist.  Or am I not understanding the semantics of that setting?

Reply:
My VSCode was marking those as incorrect without setting "required" to something.  Since the others were marked true and these were missing the value, I defaulted them to false.